qiskit/qobj/converters/pulse_instruction.py

```diff
@@ -902,7 +902,7 @@ def _convert_parametric_pulse(
         try:
             pulse_name = instruction.label
         except AttributeError:
-            sorted_params = sorted(tuple(instruction.parameters.items()), key=lambda x: x[0])
+            sorted_params = sorted(instruction.parameters.items(), key=lambda x: x[0])
```

Comment:
If we're changing this, it should really just be `sorted(instruction.parameters.keys())` (or even without the `keys()`, though I usually prefer to be explicit).

Reply:
Agree and agree.

Reply:
Fixed in 2d659c5d1c2bd0d98297063c22e09d7d26f48c38

Reply:
I undid the "fix" in 316dc91 because `sorted(instruction.parameters.keys())` does not do the same thing as the original.